src/data/general-cs-quiz.ts

```diff
@@ -1,4 +1,14 @@
 const generalCSQuiz = [
+  {
+    Question: "How many layers are there in the OSI Model in Computer Networking ?",
+    Answer: "7",
+    Distractor1: "6",
+    Distractor2: "5",
+    Distractor3: "8",
+    Explanation:
+      "There are 7 layers present in the OSI model, Application, Presentation, Session, Transport, Network, Data Link and Physical layer.",
+    Link: "https://www.youtube.com/watch?v=qiQR5rTSshw&t=27870s"
+  },
```

Comment:
we already have a similar question in the IT file
Please come up with another question